ceph-proxy/tests/target.py

```diff
@@ -60,7 +60,6 @@ def setUpClass(cls):
     def test_ceph_health(self):
         """Make sure ceph-proxy can communicate with ceph."""
         logging.info('Wait for idle/ready status...')
```

Comment:
here we log for waiting however, the actual wait call was removed.

Reply:
Done.